features/security/src/main/feature/feature.xml

```diff
@@ -528,6 +528,13 @@ org.ops4j.pax.web.default.realmname=DDF
         <bundle>mvn:ddf.security.servlet/security-servlet-whoami/${project.version}</bundle>
     </feature>
 
+    <feature name="security-servlet-web-socket" version="${project.version}"
+             description="Defines a secure web socket servlet type.">
+        <feature>security-core-impl</feature>
```

Comment:
This might need to be changed now. Try rebasing and see if this actually needs impl.

Reply:
👍 I'll see if CI fails on it first. Scared of building atm

Reply:
Manually verified it is now using artifacts from `security-core-api`. Making the change.